README.adoc

```diff
@@ -349,19 +510,71 @@ system-deployment-6bd97d9bf6-x4zth      1/1       Running   0          25s
+endif::[]
+
+// Cloud-hosted guide instruction
+ifdef::cloud-hosted[]
+Wait for your two new pods to be in the ready state, then make the following `curl` command:
+
+```
+curl -I http://$SYSTEM_HOST:31000/system/properties
+```
+{: codeblock}
+endif::[]
+
+You'll notice that the `X-Pod-Name` header will have a different value when you call it multiple times. 
+This is because there are now three pods running all serving the `system` application. 
+Similarly, to descale your deployments you can use the same scale command with fewer replicas.
+
+[role='command']
+```
+kubectl scale deployment/system-deployment --replicas=1
+```
 
 == Redeploy microservices
 
-When you're building your application, you may find that you want to quickly test a change. To do that, you can rebuild your Docker images then delete and re-create your {kube} resources. Note that there will only be one `system` pod after you redeploy since you're deleting all of the existing pods.
+When you're building your application, you may find that you want to quickly test a change. 
+To do that, you can rebuild your Docker images then delete and re-create your {kube} resources. 
+Note that there will only be one `system` pod after you redeploy since you're deleting all of the existing pods.
+
+// Static guide instruction
+ifndef::cloud-hosted[]
 [role='command']
 ```
 mvn clean package
+docker build -t system:1.0-SNAPSHOT system/.
+docker build -t inventory:1.0-SNAPSHOT inventory/.
+
+kubectl delete -f kubernetes.yaml
+kubectl apply -f kubernetes.yaml
+```
+endif::[]
+
+// Cloud-hosted guide instruction
+ifdef::cloud-hosted[]
+```
+mvn clean package
+docker build -t system:1.0-SNAPSHOT system/.
+docker build -t inventory:1.0-SNAPSHOT inventory/.
+docker tag inventory:1.0-SNAPSHOT us.icr.io/$NAMESPACE_NAME/inventory:1.0-SNAPSHOT
+docker tag system:1.0-SNAPSHOT us.icr.io/$NAMESPACE_NAME/system:1.0-SNAPSHOT
+docker push us.icr.io/$NAMESPACE_NAME/inventory:1.0-SNAPSHOT
+docker push us.icr.io/$NAMESPACE_NAME/system:1.0-SNAPSHOT
+
 kubectl delete -f kubernetes.yaml
 kubectl apply -f kubernetes.yaml
 ```
+{: codeblock}
+endif::[]
 
-This is not how you would want to update your applications when running in production, but in a development environment this is fine. If you want to deploy an updated image to a production cluster, you can update the container in your deployment with a new image. Then, {kube} will automate the creation of a new container and decommissioning of the old one once the new container is ready.
+This is not how you would want to update your applications when running in production, 
+but in a development environment this is fine. 
+If you want to deploy an updated image to a production cluster, 
+you can update the container in your deployment with a new image. 
+Then, {kube} will automate the creation of a new container and decommissioning of the old one once the new container is ready.
```

Comment:
Then, {kube} will automate the creation of a new container and decommissioning of the old one once the new container is ready.
--->
 Once the new container is ready, {kube} automates both the creation of a new container and the decommissioning of the old one.